src/utils.ts

```diff
@@ -18,6 +18,11 @@ export async function isGitInstalled(): Promise<boolean> {
   });
 }
 
+export function isFileGitignored(filePath: string): boolean {
```

Comment:
Can you add a comment to explain we know this is possible because ggshield requires git to work.